contracts/MyStrategy.sol

```diff
@@ -56,13 +59,32 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
 
     uint256 private constant BPT_WETH_INDEX = 1;
 
+    // Bribe Token -> Bribe Recepient 
+    mapping (address => address) public bribesRedirectionPaths;
+    // Bribe Recepient -> Redirection Fee
+    mapping (address => uint256) public redirectionFees;
+
     event TreeDistribution(
         address indexed token,
         uint256 amount,
         uint256 indexed blockNumber,
         uint256 timestamp
     );
     event RewardsCollected(address token, uint256 amount);
+    event RedirectionFee(
+        address indexed destination,
+        address indexed token,
+        uint256 amount,
+        uint256 indexed blockNumber,
+        uint256 timestamp
+    );
+    event TokenRedirection(
+        address indexed destination,
+        address indexed token,
+        uint256 amount,
+        uint256 indexed blockNumber,
+        uint256 timestamp
+    );
```

Comment:
It's about time we check if block number and timestamp are needed to be logged in events or if they can be inferred while querying the events

Reply:
Good call, @petrovska-petro has been exploring the standardization of revenue tracking events, maybe he can give us more insights into this.

Reply:
i am fan of adding them in the event itself since they make much easier the process of tracking those items while querying them. Even tho, we could say that if we leverage [eth_getLogs](https://www.quicknode.com/docs/ethereum/eth_getLogs) will provide the `block_number` and from it we could deduce the timestamp of that block, but will add up lots of operation to check the relation between block height -> timestamp, which each time will need to call [this](https://www.quicknode.com/docs/ethereum/eth_getBlockByNumber) into the rpc, overall making the process much slower to catch up all this information. so will advise to include them out of the box in the event itself to facilitate the querying of this events